docs/source/considerations.rst

```diff
@@ -108,3 +108,27 @@ however you may notice some differences:
    to. As the binding was executed on initialization, we need to use the
    ``addr()`` method, which will return the address associated to the alias
    passed as parameter (in the example above it is ``main``).
+
+
+Setting initial attributes
+==========================
+
+Many times, after spawning an agent, we want to set some attributes, which
+may be used to configure the agent before it starts working with the rest of
+the multi-agent system:
+
+.. code-block:: python
+
+   a0 = run_agent('foo')
+   a0.set_attr(x=1, y=2)
+
+It is such a common task that a parameter ``init`` can be used when running
+the agent for exactly that:
+
+.. code-block:: python
+
+   a0 = run_agent('foo', init=dict(x=1, y=2))
```

Comment:
It is a bit more compact and perhaps looks more similar to standard parameter values (i.e.: `function(x=1, y=2)`. Anyway the user is free to use their preferred way. When we don't agree or there is not a good reason, better leave it as-is. In this case it is just a matter of preference (I also many times prefer the use of `{}`). :stuck_out_tongue_winking_eye: